themes/opentermsarchive/assets/css/components/card.css

```diff
@@ -0,0 +1,149 @@
+.card {
+  border: 1px solid var(--colorBlack300);
+  @mixin grid 4, 3;
+  flex-shrink: 0;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+
+  &.card--center {
```

Comment:
Is it possible to have cards inside cards? Otherwise why use the direct child selector `>`?

Reply:
No, it's a [nesting rule](https://www.w3.org/TR/css-nesting-1/) so that modifier applies on the elements that have the `card` and `card--center` classes: `<div class="card card--center"></div>`

Reply:
Yes, I understand, but why augment the specification by using `& > .card__body {` instead of `& .card__body {`?
If the `card` component can't contain another `card`, I don't see the added value here.